rollout/canary.go

```diff
@@ -66,16 +66,16 @@ func (c *rolloutContext) rolloutCanary() error {
 		return err
 	}
 
-	noScalingOccurred, err := c.reconcileCanaryReplicaSets()
+	stillReconciling, err := c.reconcileCanaryReplicaSets()
```

Comment:
**Minor Suggestion:** Should we make it `isStillReconciling` or simply `isReconciling` instead?

Reply:
Updated. Thanks for the suggestion.